addons/mail/static/src/utils/common/format.js

```diff
@@ -27,7 +27,7 @@ const messageUrlRegExp = new RegExp(`^${escapeRegExp(getOrigin())}/mail/message/
  * @param {import("models").Persona[]} validMentions.partners
  * @returns {Promise<string|ReturnType<markup>>}
  */
-export function prettifyMessageText(rawBody, { validMentions = [] } = {}) {
+export function prettifyMessageText(rawBody, { validMentions = [] } = {}, thread) {
```

Comment:
Would make more sense to put `thread` in the 2nd param that acts as `options`, so that the parameter is named and it can be optional without having to deal with positional arguments (like here as positional, it sucks to force passing as 3rd param).